hardhat.config.ts

```diff
@@ -25,25 +30,38 @@ const config: HardhatUserConfig = {
     outDir: "typechain",
     target: "ethers-v6",
   },
+  // typesc
   namedAccounts: {
     deployer: {
       default: 0,
     },
+    usdc: {
+      [ChainID.Avalanche]: "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E",
+      [ChainID.Arbitrum]: "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
+    },
+    tokenMessenger: {
+      [ChainID.Avalanche]: "0x6b25532e1060ce10cc3b0a99e5683b91bfde6982",
+      [ChainID.Arbitrum]: "0x19330d10D9Cc8751218eaf51E8885D058642E08A",
+    },
+    messageTransmitter: {
+      [ChainID.Avalanche]: "0x8186359af5f57fbb40c6b14a588d2a59c0c29880",
+      [ChainID.Arbitrum]: "0xC30362313FBBA5cf9163F0bb16a0e01f01A896ca",
```

Comment:
instead of enum use string names of networks as you set them in the hardhat network config fields below